ui/app/AppLayouts/Onboarding2/KeycardCreateProfileFlow.qml

```diff
@@ -78,10 +78,14 @@ SQUtils.QObject {
         id: createKeycardProfilePage
 
         CreateKeycardProfilePage {
-            onCreateKeycardProfileWithNewSeedphrase:
-                root.stackView.push(backupSeedIntroPage)
-            onCreateKeycardProfileWithExistingSeedphrase:
-                root.stackView.push(seedphrasePage)
+            onCreateKeycardProfileWithNewSeedphrase: {
+                d.fromBackupSeedphrase = true
+                root.stackView.push(keycardCreatePinPage)
+            }
+            onCreateKeycardProfileWithExistingSeedphrase: {
+                d.fromBackupSeedphrase = false
```

Comment:
if we have backup we use existing seedphrase. In my understanding it's reversed. We need to adjust naming or describe to avoid confusion :sweat_smile:

Reply:
Fixed